### PR TITLE
AirPlace with Leftclick

### LINKED ISSUE #132: Make airplace work with worldedit pos1 selection
currently you can use airplace with the right click from the worldedit selection tool (because you're just right clicking an air block), but it would be useful to also allow the player to left click to select pos1

### PR DESCRIPTION
**Pull request by Dav321**
resolves #132

### CODE REVIEW

File: src/main/java/tools/redstone/redstonetools/mixin/features/AirplaceMixin.java [hunk 1]
```diff
@@ -23,25 +24,60 @@ public class AirplaceMixin {
...
         // If the hit result is already set
         if (crosshairTarget != null && crosshairTarget.getType() != HitResult.Type.MISS) {
-            return;
+            return false;
         }
 
-        var player = ((MinecraftClient) (Object) this).player;
-        var interactionManager = ((MinecraftClient) (Object) this).interactionManager;
+        // If the player or interactionManager not initialized
+        var minecraftClient = (MinecraftClient) (Object) this;
```
Comment (Matthias1590) [inline src/main/java/tools/redstone/redstonetools/mixin/features/AirplaceMixin.java:61]: Might be nice to abstract this into a getMinecraftClient method and then create one for getPlayer and getInteractionManager too maybe

**Review by Matthias1590 — CHANGES_REQUESTED** [review]
Might be nice to abstract a couple of lines into separate functions but the rest looks good

**Comment by Matthias1590** [comment]
Also what's your discord? So I can give you the contributor role

**Comment by Dav321** [comment]
My discord is Dav32_1#5015.

**Review by Matthias1590 — APPROVED** [review]

**Comment by Dav321** [comment]
Changed the base branch because main is protected

**Review by Matthias1590 — APPROVED** [review]

**Review by Matthias1590 — APPROVED** [review]

**Pull request merged**